src/content/guides/installing-optional-ingress-controller/index.md

```diff
@@ -14,13 +14,10 @@ An ingress controller helps you expose your services to the outside world.
 
 ## Which workload cluster releases do not ship with an ingress controller
 
-Currently clusters on our AWS and Azure installations that have a workload cluster release version
-newer than `10.0.0` (AWS) or `12.0.0` (Azure) ship without an ingress controller by default. That
-allows you full control to choose which and how many ingress controllers you
-want to run on your cluster.
+Clusters on our AWS, Azure, and KVM (On-premises) installations that have a workload cluster release version newer than `10.0.0` (AWS), `12.0.0` (Azure), and `12.2.0` (KVM) ship without an ingress controller by default.
```

Comment:
```suggestion
Clusters on our AWS, Azure, and KVM (On-premises) installations that have a workload cluster release version newer than `10.0.0` (AWS), `12.0.0` (Azure), and `12.2.0` (KVM) ship without an ingress controller by default. (Related: [Preinstalled and optional apps in workload clusters](https://docs.giantswarm.io/reference/workload-cluster-release-versions/#apps))
```

Reply:
@ubergesundheit what's the right relative link 🙈

Reply:
```suggestion
Clusters on our AWS, Azure, and KVM (On-premises) installations that have a workload cluster release version newer than `10.0.0` (AWS), `12.0.0` (Azure), and `12.2.0` (KVM) ship without an ingress controller by default. (Related: [Preinstalled and optional apps in workload clusters]({{< relref "/content/reference/workload-cluster-release-versions/_index.md#apps" >}}))
```